LoggingKata/TacoParser.cs

```diff
@@ -1,18 +1,32 @@
 ﻿namespace LoggingKata
 {
-    /// <summary>
-    /// Parses a POI file to locate all the Taco Bells
-    /// </summary>
     public class TacoParser
     {
         readonly ILog logger = new TacoLogger();
         
         public ITrackable Parse(string line)
         {
-            logger.LogInfo("Begin parsing");
 
-            // Do not fail if one record parsing fails, return null
-            return null; // TODO Implement
+            if (line == null)
+            {
+                return null;
+            }
+            TacoBell trackableTacoBell = new TacoBell();
+            Point Location = new Point();
```

Comment:
It might be better, organizationally, to instantiate these lines 14-15 down closer to where they are first used around line 24